cmake/configs/default.cmake

```diff
@@ -90,6 +90,7 @@ hunter_config(ZLIB VERSION 1.2.8-p3)
 hunter_config(ZMQPP VERSION 4.1.2)
 hunter_config(ZeroMQ VERSION 4.1.4-p2)
 hunter_config(caffe VERSION rc3-p2)
+hunter_config(autobahn-cpp VERSION 0.2.0)
```

Comment:
You should not merge your regular branches with ingenue/hunter branches. Workflow is `develop` -> `pkg.template` -> `pkg.*` -> `upload.*`. You should not "jump" from `develop` directly to `pkg.*`. The rule is simple if you're making pull request to `pkg.*` branch you have to check that it contains only `.travis.yml`/`appveyor.yml` modifications. Just like top-level README of this repository states:
* https://github.com/ingenue/hunter/